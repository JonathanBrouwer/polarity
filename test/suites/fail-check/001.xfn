codata Fun(a: Type, b: Type) : Type {
    Fun(a, b).apply(a: Type, b: Type, x: a) : b
}

impl Fun {
    codef ConvertAny(a: Type, b: Type): Fun(a, b) {
        apply(a: Type, b: Type, x: a) => x
    }
}
