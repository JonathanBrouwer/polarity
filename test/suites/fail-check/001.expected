@1.1 is not equal to @1.0